app/Platform/Actions/UpdatePlayerGameMetrics.php

```diff
@@ -37,7 +37,7 @@ public function execute(PlayerGame $playerGame, bool $silent = false): void
 
         $points = $achievementsUnlocked->sum('Points');
         $pointsHardcore = $achievementsUnlockedHardcore->sum('Points');
-        $pointsWeighted = $achievementsUnlocked->sum('TrueRatio');
+        $pointsWeighted = $achievementsUnlockedHardcore->sum('TrueRatio');
```

Comment:
fixes `points_weighted` calculation for `player_games`

Reply:
Good catch!